user_reservation.py

```diff
@@ -0,0 +1,299 @@
+            rozmiar = 4
+            monitor = False
+            return rozmiar, monitor
+
+        elif int(user_choice_size) == 10 and int(user_choice_display) == 1:
+            print("Prezentuję ofertę dostępnych biurek 10-osobowych z monitorem")
+            rozmiar = 10
+            monitor = True
+            return rozmiar, monitor
+
+        elif int(user_choice_size) == 10 and int(user_choice_display) == 2:
+            print("Prezentuję ofertę dostępnych biurek 10-osobowych bez monitora")
+            rozmiar = 10
+            monitor = False
+            return rozmiar, monitor
+    size, display = presenting_avaliability()
+
+    def checking_availiability_by_date(filename='user_data.json', user_beginning_date_str='', user_end_date_str=''):
+        with open(filename, 'r') as file:
+            data = json.load(file)
+
+        user_beginning_date = datetime.strptime(user_beginning_date_str, "%Y-%m-%d")
+        user_end_date = datetime.strptime(user_end_date_str, "%Y-%m-%d")
+
+        reserved_desks = []
+
+        for id_reservation, reservation_spec in data.items():
+            reservation_start = datetime.strptime(reservation_spec['Wynajem od'], "%Y-%m-%d")
+            reservation_end = datetime.strptime(reservation_spec['Wynajem do'], "%Y-%m-%d")
+
+            if not (user_end_date <= reservation_start or user_beginning_date >= reservation_end):
+                reserved_desks.append(reservation_spec['Biurko'])
+
+        return reserved_desks
+    reserved_desks = checking_availiability_by_date(user_beginning_date_str=user_beginning_date_str, user_end_date_str=user_end_date_str)
+
+    def choosing_desk_v2(exclude_desk_ids=None, size=None, display=None):
+        with open('desks.json', 'r') as file:
+            data = json.load(file)
+
+        if exclude_desk_ids is None:
+            exclude_desk_ids = []
+
+        for desk_name, desk_info in data.items():
+            if desk_name not in exclude_desk_ids and (size == desk_info['rozmiar'] and display == desk_info['monitor']):
+                print(f"Numer biurka: {desk_name}:")
+                print(f"Specyfikacja: {desk_info['desk_type']}, "
+                      f"\nCena za jedno stanowisko: {desk_info['price']} PLN")
+                if desk_info['rozmiar'] > 1:
+                    print(f"Całkowita cena wynajmu biurka: {desk_info['price'] * desk_info['rozmiar']}")
+                print("")
+            else:
+                continue
+    choosing_desk_v2(exclude_desk_ids=reserved_desks, size=size, display=display)
+
+    print("Oto lista dostępnych biurek.")
+    user_desk_choice = input("Wprowadź numer wybranego przez siebie biurka: ")
+
+    def choosing_desk_from_displayed(user_desk_choice):
+        with open('desks.json', 'r') as file:
+            data = json.load(file)
+
+        user_desk_choice = str(user_desk_choice)
+
+        for desk_name, desk_info in data.items():
```

Comment:
po co petla skoro mozesz zrobic po prostu
```suggestion
        choosed_desk = data.get(user_desk_choice)
```